PWD_Vault/PWD_Vault.py

```diff
@@ -1,86 +1,88 @@
+    fn_l = [one, two, three, four, five, six, seven, eight, nine, ten, eleven, twelve, thirteen, fourteen, fifteen, sixteen, seventeen, eighteen, nineteen]
     random.choice(fn_l)()
 
 def login_screen():
     global key
     try:
-
-        print('|-------------------------------------------------------|')
-        print('|--------------------LOGIN SCREEN ----------------------|')
-        print('|-------------------------------------------------------|')
-        print('|----1. Enter the Master Password ----------------------|')
-        print('|-------------------------------------------------------|')
-        print('|-------------------------------------------------------|')
-        print('|-------------------------------------------------------|')
-        print('|--s = First-Time-Setup (Only run this once) -----------|')
-        print('|-------------------------------------------------------|')
-        print('|-------------------------------------------------------|')    
-        print('|----h = Help ------------------------------------------|')
-        print('|----q = quit ------------------------------------------|')
-        print('|-------------------------------------------------------|')
-        print('\n')
-
-
+        
+        if not os.path.exists('salt.dat') or not os.path.exists('m_pwd.dat'):
+            print("Run the first-time-setup first (option 's')...")
+            login_screen()
+            return 
+
+        login_interface = """
+        |-------------------------------------------------------|
+        |--------------------LOGIN SCREEN ----------------------|
+        |-------------------------------------------------------|
+        |----1. Enter the Master Password ----------------------|
+        |-------------------------------------------------------|
+        |-------------------------------------------------------|
+        |-------------------------------------------------------|
+        |-------------------------------------------------------|
+        |-------------------------------------------------------|    
+        |----h = Help ------------------------------------------|
+        |----q = quit ------------------------------------------|
+        |-------------------------------------------------------|
+        """
+        print (login_interface)
 
         option = input('> ')
-        if option == '1':
-            key = check_encrypted_password()
-            
-            if endecrypt.check_pwd == True:
-                print('Login Success!')
-                decrypt(key)
-                main_menu()
-            else: 
-                print('Wrong Password!!')
-                login_screen()
+        
+        match option:
+            case'1':
+                key = check_encrypted_password()
             
-        elif option == 's':

```

Comment:
You removed the functionality to run the first time setup but where did you add it back again?

Reply:
I actually added it inside the "Master Key" option, where in case you it's your first time entering the master key then it will automatically redirect you to "first time set-up".  The main reason I did this change was because earlier I was facing a bug where a user could create multiple master-key as option == "s" was still accessible on the login-interface.

Please do check and let me know the changes I can implement on it.

Reply:
I did that because I would accidently just close the terminal while logged in. So in order the encrypt the file again I would need to run the first time setup again. 
Multiple master keys should not be possible as the first-time setup replaces the original `salt.dat` with the new `salt.dat`